CDP4Composition/Services/AppSettingService/PluginSettingsMetaData.cs

```diff
@@ -0,0 +1,101 @@
+﻿// -------------------------------------------------------------------------------------------------
+// <copyright file="PluginSettingsMetaData.cs" company="RHEA System S.A.">
+//   Copyright (c) 2018-2020 RHEA System S.A.
```

Comment:
@samatrhea I'm used to only use the current year for new files:

```//   Copyright (c) 2020 RHEA System S.A.```

I don't really care how we do it, as long as we all do it the same way.

Reply:
@alexatrhea or @samatrhea can you confirm? I checked over solution and many are using 2015-2020.

Reply:
both are fine as long as 2020 is there